src/placeos-models/asset.cr

```diff
@@ -0,0 +1,54 @@
+require "./base/model"
+
+module PlaceOS::Model
+  class Asset < ModelBase
+    include RethinkORM::Timestamps
+
+    table :asset
+
+    attribute name : String, es_subfield: "keyword"
+    attribute category : String = ""
+    attribute description : String = ""
+    attribute size : String = ""
```

Comment:
This is a mega vague field. I know it's part of the spec, but could you add a comment that we need to refine this field.

We can deal with it down the line if it's in-use/not in-use